news/_posts/2021-03-02-archived-modules.md

```diff
@@ -0,0 +1,30 @@
+title:  "Focusing on essential modules"
+subtitle: "Announcing end of official support for less popular modules"
+date:   2021-03-02 14:00:00
+authors:  [ MarionFrancois ]
+icon: icon-puzzle
+tags: 
+ - modules
+ - news
+---
+
+The product team recently did a survey of almost 600 PrestaShop users to find out which modules were essential for their business and which ones they used the least.
+
+Following this survey, the product team and the developer team decided to maintain a smaller set of modules that are really important for the users, rather than attempting to keep all modules afloat which results in a poor maintenance of each and a constant struggle to update everything.
+
+The maintainers team resources are limited, and there are too many free and open source modules on the PrestaShop GitHub organization to be able to maintain them all. Because of this maintenance burden, some of these modules have not been given a new release for years.
+
+This is why the following set of modules, reported in the survey among the least used have been archived: [statslive](https://github.com/PrestaShop/statslive), [ps_advertising](https://github.com/PrestaShop/ps_advertising), [ps_feeder](https://github.com/PrestaShop/ps_feeder), [ps_legalcompliance](https://github.com/PrestaShop/ps_legalcompliance), [ps_productinfo](https://github.com/PrestaShop/ps_productinfo), [ps_rssfeed](https://github.com/PrestaShop/ps_rssfeed), [pscleaner](https://github.com/PrestaShop/pscleaner), [trackingfront](https://github.com/PrestaShop/trackingfront), [watermark](https://github.com/PrestaShop/watermark), [prestafraud](https://github.com/PrestaShop/prestafraud), [gapi](https://github.com/PrestaShop/gapi).
+
+The following modules have also been archived but were already not available for the 1.7 stores because they are incompatible: [dateofdelivery](https://github.com/PrestaShop/dateofdelivery), [newsletter](https://github.com/PrestaShop/newsletter), [vatnumber](https://github.com/PrestaShop/vatnumber), [ps_carriercomparison](https://github.com/PrestaShop/ps_carriercomparison), [referralprogram](https://github.com/PrestaShop/referralprogram), [ps_healthcheck](https://github.com/PrestaShop/ps_healthcheck), [loyalty](https://github.com/PrestaShop/loyalty).
+
+
+The repositories of these modules have been archived. This means that they will not be available in back-offices, but they are still available on GitHub. No further work will happen on these GitHub repositories, but developers can still fork them and use them.
+But here's the upside: PrestaShop is an open source, community project. If the PrestaShop project stops maintaining a repository, it does not mean this project is gone. What the project cannot do by itself, the community can do it. We might see contributors fork these modules and build a newer, better version that will surpass the current module.
+Of course, you can also take a look at the marketplace where you could find similar modules to the archived ones.
```

Comment:
```suggestion

Of course, you can also take a look at the marketplace where you could find similar modules to the archived ones.
```
If we dont add the blank line, it will be displayed as a block.